app/helpers/view_helper.rb

```diff
@@ -0,0 +1,47 @@
+module ViewHelper
```

Comment:
I just wonder; if these apply for the whole application, and they look like they do, then perhaps they should go in application helper.

Reply:
I think the concerns of various link helpers shouldn't really be used outside views - they talk about css, html and js.

I'd start by keeping them in the view helper, and if/when we have a use case for using them in a wider scope, we can move them.